pkg/controllers/elasticquota_controller_test.go

```diff
@@ -203,25 +203,25 @@ func TestElasticQuotaController_Run(t *testing.T) {
 					t.Errorf("reconcile: (%v)", err)
 				}
 			}
-
-			err := wait.Poll(200*time.Millisecond, 1*time.Second, func() (done bool, err error) {
-				for _, v := range c.want {
-					eq := &v1alpha1.ElasticQuota{
-						ObjectMeta: metav1.ObjectMeta{
-							Name:      v.Name,
-							Namespace: v.Namespace,
-						},
-					}
-					err := kClient.Get(ctx, client.ObjectKeyFromObject(eq), eq)
-					if err != nil {
-						return false, err
+			err := wait.PollUntilContextTimeout(ctx, 200*time.Millisecond, 1*time.Second,
+				false, func(ctx context.Context) (done bool, err error) {
```

Comment:
can we keep the function signature in just one (admittedly long) line? that would make the review much easier because only that line should change

Reply:
Here and everywhere else

Reply:
Okay, thanks for reminding this, it helps the readability of the review. done!